docs/index.rst

```diff
@@ -12,10 +12,10 @@ urllib3.future
    contributing
    changelog
 
-⚡ urllib3.future is a powerful, *user-friendly* HTTP client for Python.
-⚡ urllib3.future goes beyond supported features while remaining compatible.
+- ⚡ urllib3.future is a powerful, *user-friendly* HTTP client for Python.
```

Comment:
Remove the emojis. They look kitchy.
Compare semiconductor documentation to get a sense of non-distracting professional  presentation/writing style..